contributing_to_docs/term_glossary.adoc

```diff
@@ -84,6 +89,13 @@ Usage: build(s) as appropriate
 
 ''''
 
+=== cluster
+
+Usage: cluster
+
+The collection of controllers, pods, and services and related DNS and networking
+routing configuration that are defined on the system.
+''''
 === container
```

Comment:
There are a lot of places where we have a '''' line adjacent to an === line.  Shall I fix all such instances?